### PR TITLE
Add LTS tag and push manifest lists for LTS

### PR DESCRIPTION
**Pull request by anamnavi**
## PR Summary

Generate manifest lists for LTS channel. Update meta.json for the LTS channel images as well.

WIP because I'm not yet fully sure if I used the right values for the manifestList key/value in the meta.json files.

Addresses #527 

## PR Checklist

- [x] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
  - Use the present tense and imperative mood when describing your changes
- [x] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [x] [Change is not breaking](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)
- [x] [Make sure all `Dockerfile`, `.sh`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
  - If the PR is work in progress, please add the prefix `WIP:` to the beginning of the title and remove the prefix when the PR is ready.
- **Adds a new image or tag**
  - [ ] Not Applicable
- **OR**
  - [ ] Update [README.powershellcommunity.md](https://github.com/PowerShell/PowerShell-Docker/blob/master/docs/README.powershellcommunity.md)
  - [ ] Update [vsts-ci.yml](https://github.com/PowerShell/PowerShell-Docker/blob/master/vsts-ci.yml)

### CODE REVIEW

File: release/lts/nanoserver2004/meta.json [hunk 1]
```diff
@@ -11,6 +11,9 @@
         "lts-nanoserver-#shorttag#",
         "lts-nanoserver-2004-#fulltag#"
     ],
+    "manifestLists": [
+        "lts"
```
Comment (TravisEz13) [inline release/lts/nanoserver2004/meta.json:15]:
```suggestion
        "nanoserver-lts"
```

File: release/lts/nanoserver1909/meta.json [hunk 2]
```diff
@@ -11,6 +11,9 @@
         "lts-nanoserver-#shorttag#",
         "lts-nanoserver-1909-#fulltag#"
     ],
+    "manifestLists": [
+        "lts"
```
Comment (TravisEz13) [inline release/lts/nanoserver1909/meta.json:15]:
```suggestion
        "nanoserver-lts"
```

**Review by TravisEz13 — CHANGES_REQUESTED** [review]

File: release/lts/nanoserver1909/meta.json (hunk 2 above)
Comment (TravisEz13) [reply release/lts/nanoserver1909/meta.json:15]: nano server shouldn't be in the same manifest list.  let's make lts the same a latest and make a new one called nanoserver-lts that is the equivalent of nanoserver

Comment (anamnavi) [reply release/lts/nanoserver1909/meta.json]: I don't quite fully understand the second sentence, but I'll ask about this in the call on Tues. I changed the manifest list values for nanoserver1809, 1909, 2004.

**Review by TravisEz13 — APPROVED** [review]

**Pull request merged**